Firestore/Swift/Source/Codable/DocumentID.swift

```diff
@@ -15,6 +15,10 @@
  */
 
 import FirebaseFirestore
+import FirebaseSharedSwift
+
+let documentRefUserInfoKey =
+  CodingUserInfoKey(rawValue: "DocumentRefUserInfoKey")!
```

Comment:
nit: Can you make this a convenience init or static method on CodingUserInfoKey so consumers don't need to know a magic string and then force-unwrap the result?

Reply:
@morganchen12 @andrewheard @mortenbekditlevsen @ncooke3 I'm going to resolve in order to merge today, but would appreciate a resolution/ follow up PR in the next few days